pipeline/coaddition.py

```diff
@@ -0,0 +1,423 @@
+
+import numpy as np
+from numpy.fft import fft2, ifft2, fftshift
+
+from sep import Background
+
+from models.provenance import Provenance
+from models.image import Image
+
+from pipeline.parameters import Parameters
+
+from improc.bitmask_tools import dilate_bitmask
+from improc.inpainting import Inpainter
+from improc.tools import sigma_clipping
+
+
+class ParsCoadd(Parameters):
+    def __init__(self, **kwargs):
+        super().__init__()
+
+        self.method = self.add_par(
+            'method',
+            'zogy',
+            str,
+            'Coaddition method.  Currently only "naive" and "zogy" are supported. ',
```

Comment:
We should add an issue to add the swarp coaddition method.  (In that case, alignment and coaddition can be done as a single step.)

Reply:
Issue #184